jbpm-designer-backend/src/test/java/org/jbpm/designer/repository/VFSRepositoryGitFileSystemTest.java

```diff
@@ -965,9 +968,13 @@ public void testMoveEmptyDirectory() throws NoSuchFileException {
     }
 
     @Test
-    public void testCreateGlobalDirOnNewProject() throws FileAlreadyExistsException {
+    public void testCreateGlobalDirOnNewProject() throws Exception {
+        ServletContext servletContext = mock(ServletContext.class);
+        when (servletContext.getRealPath(anyString())).thenReturn(getClass().getResource("default.json").getFile());
+
         VFSRepository repository = new VFSRepository(producer.getIoService());
         repository.setDescriptor(descriptor);
+        repository.setServletContext(servletContext);
```

Comment:
I am wondering why there was no need for more changes in the test, suggest me that the change is not tested properly. Or do the existing tests cover the change already?

Reply:
I honestly think the test is fine. The change in the method tested is still fully tested (just using servletcontext to get the default files rather than the hardcoded default sources).